src/booleans.js

```diff
@@ -1,62 +1,94 @@
 function negate(a) {
   // your code here
+  return !(a > 0);
 };
 
 function both(a, b) {
   // your code here
+  return a > 0 && b > 0;
 };
 
 function either(a, b) {
   // your code here
+  return a > 0 || b > 0;
 };
 
 function none(a, b) {
   // your code here
+  return !(a > 0 || b > 0);
 };
 
 function one(a, b) {
   // your code here
+  if (a > 0 && b > 0) {
+    return false;
+  }
+  return a > 0 || b > 0;
```

Comment:
You can simplify this to remove the if statement, and refactor this just slightly:
```
  return (a && !b) || (!a && b);
```